specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-02-02-preview/examples/ManagedClusterSnapshotsList.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parameters": {
+    "api-version": "2022-02-02-preview",
+    "subscriptionId": "subid1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "snapshot1",
+            "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/managedclustersnapshots/snapshot1",
+            "type": "Microsoft.ContainerService/ManagedClusterSnapshots",
+            "location": "westus",
+            "tags": {
+              "key1": "val1",
+              "key2": "val2"
+            },
+            "systemData": {
+              "createdBy": "user1",
+              "createdByType": "User",
+              "createdAt": "2021-08-09T20:13:23.298420761Z"
+            },
+            "properties": {
```

Comment:
>properties [](http://example.com/codeflow?start=13&length=10)

I noticed that managedClusterPropertiesReadOnly is missing in the examples for List APIs.

Not sure it is just missing from the example or the api actually does not return the property.

List must return the same schema as GET/PUT/PATCH.
[https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc008-put-get-patch--list-must-return-the-same-resource-schema](https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc008-put-get-patch--list-must-return-the-same-resource-schema)

Can you please update the examples to include the missing property?